cluster-autoscaler/cloudprovider/aws/aws_manager.go

```diff
@@ -49,7 +49,7 @@ const (
 	operationPollInterval   = 100 * time.Millisecond
 	maxRecordsReturnedByAPI = 100
 	maxAsgNamesPerDescribe  = 50
-	refreshInterval         = 1 * time.Minute
+	refreshInterval         = 10 * time.Second
```

Comment:
This was changed within out 1.12 branch (I assume). Do we want to keep it?

Reply:
No https://github.com/zalando-incubator/autoscaler/commit/6f50476a7a372b267f3fac43e7f130fab6eb05ad

TODO: Use 1 minute here